package.json

```diff
@@ -29,7 +29,7 @@
     "prettier:write": "prettier --write 'docs/**/*.md' README.md '.github/**' package.json tsconfig.json src/globals.json .yarnrc.yml",
     "postpublish": "pinst --enable",
     "test": "jest",
-    "tools:regenerate-docs": "ts-node -T tools/regenerate-docs",
+    "tools:regenerate-docs": "yarn prepack && eslint-doc-generator --ignore-config all --rule-doc-title-format desc-parens-name --rule-doc-section-include \"Rule details\" --rule-list-columns name,description,configsError,configsWarn,configsOff,fixable,hasSuggestions,deprecated --split-by meta.docs.requiresTypeChecking --url-configs \"https://github.com/jest-community/eslint-plugin-jest/blob/main/README.md#shareable-configurations\" && yarn prettier:write",
```

Comment:
Can I vote again for having a config file? 😅

Reply:
Haha yeah a config file (https://github.com/bmish/eslint-doc-generator/issues/61) will be one of my top priorities now that I've successfully introduced this tool into ~10 top ESLint plugins (a month of work was needed to implement all the baseline functionality necessary for that).

Reply:
Implemented: https://github.com/bmish/eslint-doc-generator/releases/tag/v0.20.0